iroha_client_cli/src/main.rs

```diff
@@ -390,6 +428,44 @@ mod maintenance {
 
         const CONFIGURATION_PATH: &str = "tests/test_config.json";
 
+        #[async_std::test]
+        async fn cli_check_health_should_work() {
+            task::spawn(async {
```

Comment:
Let's move the duplicated code in a separate function

Reply:
@vmarkushin I’m against this. 
It will be possible in a good way after the next PR with BDD and cucumber, but right now it will split the complete test into several functions

Reply:
@humb1t interesting. Ok, let's do it in another PR